specification/mediaservices/resource-manager/Microsoft.Media/stable/2021-11-01/Accounts.json

```diff
@@ -0,0 +1,1820 @@
+        },
+        "storageAuthentication": {
+          "type": "string",
+          "enum": [
+            "System",
+            "ManagedIdentity"
+          ],
+          "x-ms-enum": {
+            "name": "StorageAuthentication",
+            "values": [
+              {
+                "value": "System",
+                "description": "System authentication."
+              },
+              {
+                "value": "ManagedIdentity",
+                "description": "Managed Identity authentication."
+              }
+            ],
+            "modelAsString": true
+          },
+          "x-nullable": true
+        },
+        "encryption": {
+          "$ref": "#/definitions/AccountEncryption",
+          "description": "The account encryption properties."
+        },
+        "keyDelivery": {
+          "$ref": "#/definitions/KeyDelivery",
+          "description": "The Key Delivery properties for Media Services account."
+        },
+        "publicNetworkAccess": {
+          "type": "string",
+          "enum": [
+            "Enabled",
+            "Disabled"
+          ],
+          "x-ms-enum": {
+            "name": "PublicNetworkAccess",
+            "values": [
+              {
+                "value": "Enabled",
+                "description": "Public network access is enabled."
+              },
+              {
+                "value": "Disabled",
+                "description": "Public network access is disabled."
+              }
+            ],
+            "modelAsString": true
+          },
+          "description": "Whether or not public network access is allowed for resources under the Media Services account.",
+          "x-nullable": true
+        },
+        "provisioningState": {
+          "type": "string",
+          "enum": [
+            "Failed",
+            "InProgress",
+            "Succeeded"
+          ],
+          "x-ms-enum": {
+            "name": "ProvisioningState",
+            "values": [
+              {
+                "value": "Failed",
+                "description": "Provisioning state failed."
+              },
+              {
+                "value": "InProgress",
+                "description": "Provisioning state in progress."
+              },
+              {
+                "value": "Succeeded",
+                "description": "Provisioning state succeeded."
+              }
+            ],
+            "modelAsString": true
+          },
+          "description": "Provisioning state of the Media Services account.",
+          "readOnly": true,
+          "x-nullable": false
+        },
+        "privateEndpointConnections": {
+          "items": {
+            "$ref": "../../../../../common-types/resource-management/v2/privatelinks.json#/definitions/PrivateEndpointConnection"
+          },
+          "description": "The Private Endpoint Connections created for the Media Service account.",
+          "readOnly": true
+        }
```

Comment:
Miss the type=array here.

This will cause misbehavior on autorest.modelerfour which affects all SDK.